aten/src/ATen/native/cuda/AveragePool2d.cu

```diff
@@ -232,39 +232,33 @@ __global__ void avg_pool2d_backward_out_cuda_frame_nhwc(const int nthreads,
 
 } // anonymous namespace
 
-TORCH_IMPL_FUNC(avg_pool2d_out_cuda) (
-  const Tensor& input_,
-  IntArrayRef kernel_size,
-  IntArrayRef stride,
-  IntArrayRef padding,
-  bool ceil_mode,
-  bool count_include_pad,
-  c10::optional<int64_t> divisor_override,
-  const Tensor& output
-) {
+TORCH_IMPL_FUNC(avg_pool2d_out_cuda)
+(const Tensor& input_,
+ int kH,
+ int kW,
+ int dH,
+ int dW,
+ int padH,
+ int padW,
```

Comment:
you gotta update these to int64_t, and then safe downcast them to int in the bottom

Reply:
Yep, I was going to do this next and didn't request review on purpose for that reason (and to see what else failed on CUDA).